chpl/chpl-service/src/main/java/gov/healthit/chpl/upload/certifiedProduct/CertifiedProductHandler.java

```diff
@@ -297,7 +297,32 @@ protected PendingCertificationResultEntity getCertificationResult(String criteri
         result.setMeetsCriteria(asBoolean(columnValue));
         return result;
     }
+    
+    protected Boolean asBooleanEmpty(String value) {
+        value = value.trim();
+
+        if (StringUtils.isEmpty(value)) {
+            return null;
+        }
```

Comment:
Should this return `false`? I thought we have the theory that blank counts as false, in general.

Reply:
This is a new method that is only used to parse G1 and G2 success fields right now, for those fields they are required to enter something, either 0 or 1. For us to detect whether or not they entered something we had to create this new method that set empty to null so that we can detect if those fields are actually filled in and not just set that field to false.